### ISSUE TITLE
How to make classification for test dataset ?

### ISSUE THREAD

**Issue by hellozgm** [firstmover/ai_challenger_scene#2]
It seems there is no test script to generate JSON file named "submit.json".

**Comment by firstmover** [comment]
No, there is not. I ask my partner to do the submitting job for me. I guess it's not hard to write one if you finish evaluating test set. I think I will add this part of code soon.
